lib/matplotlib/tests/test_axes.py

```diff
@@ -2776,6 +2776,47 @@ def test_hist_stacked_normed():
     ax.hist((d1, d2), stacked=True, normed=True)
 
 
+@image_comparison(baseline_images=['hist_stacked_normed'], extensions=['png'])
+def test_hist_stacked_density():
+    # make some data
+    d1 = np.linspace(1, 3, 20)
+    d2 = np.linspace(0, 10, 50)
+    fig = plt.figure()
+    ax = fig.add_subplot(111)
+    ax.hist((d1, d2), stacked=True, density=True)
+
+
+def test_hist_normed_density():
+    # Normed and density should not be used simultaneously
+    d1 = np.linspace(1, 3, 20)
+    d2 = np.linspace(0, 10, 50)
+    fig = plt.figure()
+    ax = fig.add_subplot(111)
+    # test that kwargs normed and density cannot be set both.
+    with pytest.raises(Exception):
+        ax.hist((d1, d2), stacked=True, normed=True, density=True)
+
+    with pytest.raises(Exception):
+        ax.hist((d1, d2), stacked=True, normed=False, density=True)
+
+    with pytest.raises(Exception):
+        ax.hist((d1, d2), stacked=True, normed=False, density=False)
+
+    with pytest.raises(Exception):
+        ax.hist((d1, d2), stacked=True, normed=True, density=False)
+
+
+def test_hist_stacked_density():
```

Comment:
This test was being covered by one of the other added ones anyway, so I've just got rid of it.